articles/install-guide/index.md

```diff
@@ -23,6 +23,7 @@ In addition, the QDK provides Q# support for Jupyter Notebooks with the IQ# Jupy
 
 The QDK is available for multiple development environments. Select your preferred setup from the sections below:
 
+- [Q# standalone executable]() choose this environment if you want to run Q# on its own without a driver and using a host program like the below options.
```

Comment:
I'd suggest here as well that Q# standalone executables aren't an environment per se, so much as the compiler feature that users exploit to work with Q# from the command line.